src/actions/vm.js

```diff
@@ -76,6 +76,15 @@ export function suspendVm ({ vmId }) {
   }
 }
 
+export function editVm ({ vm }) {
```

Comment:
Please rename it to `vmId`. Javascript is not a typed, it took me a while to realize it's not an object.